install.py

```diff
@@ -24,11 +24,33 @@
 import subprocess
 import sys
 from distutils import sysconfig
+from typing import Callable, Optional, Tuple
 
 # Flush output on newlines
 sys.stdout.reconfigure(line_buffering=True)
 
 
+class bcolors:
+    OKGREEN = "\033[92m"
+    WARNING = "\033[93m"
+    FAIL = "\033[91m"
+    ENDC = "\033[0m"
```

Comment:
Could imitate the API I've used elsewhere for consistency

https://github.com/nv-legate/legate.core/blob/5a73122b7516f6b8e786d0c649522223aedc20fe/legate/driver/ui.py#L67-L68

(just manually, w/o using `colorama`)